assembly/gradle-plugin/pom.xml

```diff
@@ -175,6 +175,33 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>ci</id>
```

Comment:
I would like to see a comment here, what this profile is for and why it deactivates `--no-daemon`